packages/snfoundry/scripts_js/deploy.js

```diff
@@ -1,4 +1,10 @@
 const { deployContract } = require("./deploy_contract");
+const argv = require("yargs/yargs")(process.argv.slice(2)).argv;
+const networks = require("./helpers/networks");
+const networkName = argv.network;
+const { deployer } = networks[networkName];
+const last_devnet_account = "0x4b3f4ba8c00a02b66142a4b1dd41a4dfab4f92650922a3280977b0f03c75ee1";
+const deployer_address = deployer.address || last_devnet_account
```

Comment:
We don't need all this lines, we need to export deployer from `deploy_contract.js` and import it here similar as `deploy_contract`